.storybook/main.js

```diff
@@ -75,4 +75,8 @@ module.exports = {
     options.cache && (options.cache.set = () => Promise.resolve())
     return config
   },
+  typescript: {
```

Comment:
enables listing all properties within storybook, did not list extended interfaces otherwise